src/main/java/task/Task.java

```diff
@@ -0,0 +1,97 @@
+package main.java.task;
+
+import java.time.format.DateTimeFormatter;
+
+/**
+ * A class that encapsulates a general Task stored by Duke.
+ */
+public class Task {
+
+    protected String task;
+    protected boolean state;

```

Comment:
Consider renaming this. Booleans should be phrased to sound like booleans. Perhaps isDone might be a better name?

Reply:
Good suggestion. Thanks!